admin_guide/idling_applications.adoc

```diff
@@ -54,27 +54,39 @@ $ oc idle <service>
 
 Idle multiple services by creating a list of the desired services, then using the `--resource-names-file` option with the `oc idle` command.
 
-This is helpful if an application spans across a set of services, or when idling
-multiples services in conjunction with a script in order to idle applications in
-bulk.
+This is helpful if an application spans across a set of services within a project,
+or when idling multiple services in conjunction with a script in order to idle
+multiple applications in bulk within the same project.
 
-. Create a file containing a list of the services, each on their own line.
+. Create a simple text file containing a list of the services, each on their own line.
 
 . Idle the services using the `--resource-names-file` option:
 +
 ----
 $ oc idle --resource-names-file <filename>
 ----
 
+[NOTE]
+====
+The idle command is limited to a single project. Idling applications across a
+cluster requires the idle command to be run separately against each project.
+====
+
 [[idling-applications-unidling-applications]]
 == Unidling Applications
 
 Application services become active again when they receive network traffic and
 will be scaled back up their previous state. This includes both traffic to the
 services and traffic passing through routes.
 
+Applications may be manually unidled by scaling up the resource. Scaling up a
+deploymentconfig can be done with the following command:
```

Comment:
Instead of `Scaling up a deploymentconfig can be done with the following command:` I would write `For example, to scale up a deploymentconfig, run the command:`